src/main/webapp/index.jsp

```diff
@@ -30,6 +30,7 @@
     <% } %>
     <a href="/conversations">Conversations</a>
     <a href="/about.jsp">About</a>
+    <a href="/profile">Profile</a>
```

Comment:
Same as prev comment, order of pages in header

Reply:
fixed